nimbleclient/v1/restclient.py

```diff
@@ -203,30 +228,40 @@ def get(self, endpoint, **params):
             paginated_data.extend(response.json()["data"])
             total_rows = response.json()['totalRows']
             retrieved_rows = len(paginated_data)
-            #current_end_row = response.json()['endRow']
+            # current_end_row = response.json()['endRow']
```

Comment:
Several things here:
* You're making multiple calls to `response.json()` to retrieve different attributes. Each one of those calls is parsing the entire body of the response and turning it into JSON. So if you have a large list of items, you're paring the raw text 3 times over. You should instead put the json in a variable, you can even replace the response variable since you don't really need it anymore: `response = response.json()`, that way you can just index into response directly with `response['data']`, `response['totalRows']`, etc.
* There's no value in creating `paginated_data` as an empty list only to immediately extend it with another list. Simply define it that way from the start: `paginated_data = response['data']`

Reply:
Agreed, fixed accordingly.